styles.css

```diff
@@ -1084,26 +975,14 @@ div.dt-button-background {
     left: 0;
     width: 100%;
     height: 100%;
-    background: rgba(0, 0, 0, 0.7);
-    /* Fallback */
-    background: -ms-radial-gradient(center, ellipse farthest-corner, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
-    /* IE10 Consumer Preview */
-    background: -moz-radial-gradient(center, ellipse farthest-corner, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
-    /* Firefox */
-    background: -o-radial-gradient(center, ellipse farthest-corner, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
-    /* Opera */
-    background: -webkit-gradient(radial, center center, 0, center center, 497, color-stop(0, rgba(0, 0, 0, 0.3)), color-stop(1, rgba(0, 0, 0, 0.7)));
-    /* Webkit (Safari/Chrome 10) */
-    background: -webkit-radial-gradient(center, ellipse farthest-corner, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
-    /* Webkit (Chrome 11+) */
     background: radial-gradient(ellipse farthest-corner at center, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
     /* W3C Markup, IE10 Release Preview */
     z-index: 2001;
 }
 
 @media screen and (max-width: 640px) {
     div.dt-buttons {
-        float: none !important;
+        float: none;
```

Comment:
a lot of this going on. why?

Reply:
according to the moodle coding styleguide (https://docs.moodle.org/dev/Element_HTML_and_CSS_guidelines) : "Never use *!important* in your CSS unless it is absolutely impossible to avoid doing so.".

One consequently gets lots of stylelint errors with these (https://stylelint.io/user-guide/rules/declaration-no-important/).